servicex/models.py

```diff
@@ -179,6 +181,8 @@ def to_json(cls, x):
     generated_code_cm = db.Column(db.String(128), nullable=True)
     status = db.Column(db.String(128), nullable=True)
     failure_description = db.Column(db.String(max_string_size), nullable=True)
+    app_version = db.Column(db.String(64), nullable=True)
+    code_gen_image = db.Column(db.String(256), nullable=True)
```

Comment:
I guess the idea here is that it's possible these values will not be the same for every request in the database if the database has been migrated from some previous version of ServiceX.

Reply:
Correct, a suggestion from Gordon to insure reproducibility